src/tasks/checkEnvironment.js

```diff
@@ -46,10 +56,21 @@ var self = {
     }
   },
 
+  dockerMachineExists: function () {
+    try {
+      which('docker-machine');
+      return true;
+    } catch (err) {
+      return false;
+    }
+  },
+
   isDockerRunning: function () {
     var running;
     if (['win32', 'darwin'].indexOf(process.platform) > -1) {
-      var stdout = execSync('boot2docker status', { encoding: 'utf-8' });
+      var dockerMachineDefault = 'default';
```

Comment:
It could be.  I extracted it as a variable because we might want to provide an override.  It's possible that someone has `local`, or some other named vbox instance.